Include/Ashkal/Ashkal/Math.hpp

```diff
@@ -0,0 +1,10 @@
+#ifndef ASHKAL_MATH_HPP
+#define ASHKAL_MATH_HPP
+#include <cmath>
+
+namespace Ashkal {
+
```

Comment:
Document this and define it as:

static constexpr double PI = 3.14159265358979323;

Don't calculate it.